grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -110,7 +110,7 @@ class InvoiceItem implements Serializable {
         quantityPerUom(nullable: false)
         amount(nullable: true)
         unitPrice(nullable: true)
-        inverse(nullable: true)
+        inverse(nullable: false)
```

Comment:
@awalkowiak the migration script used to be looping through all existing invoice items and marking them as `inverse = false` which I realize is the same as doing this and since the code will probably break if this value isn't set, then I think this makes sense to make as non-null.